Gem/Code/Source/Components/UI/MatchOverComponent.cpp

```diff
@@ -65,6 +69,10 @@ namespace MultiplayerSample
             m_roundTimerHandler = AZ::EventHandler<RoundTimeSec>([this](RoundTimeSec value) 
                 { DetermineIfMatchEnded(value); });
             netMatchComponent->RoundTimeAddEvent(m_roundTimerHandler);
+
```

Comment:
Fyi, as it is written you are signing up for events on both clients and the server. You can limit the scope by using `IsNetEntityRoleClient`.

Reply:
this component is not networked though, so I don't think it has access to that API?

Reply:
Oh, yeah. You are right.